### ISSUE TITLE
"Extension server error: Object not found: <top>"

### ISSUE THREAD

**Issue by loopmode** [electron-userland/electron-webpack-quick-start#56]
The `main/index.js` file is attempting to open the devtools too soon: https://github.com/electron-userland/electron-webpack-quick-start/blob/master/src/main/index.js#L19-L21

This results in an error message printed to the console: `"Extension server error: Object not found: <top>"`

A couple of solutions are proposed in https://github.com/SimulatedGREG/electron-vue/issues/389

**Comment by thoni56** [comment]
Yes, wrapping the opening of devtools inside `did-frame-finish-load()` worked for me.

**Comment by zivyuan** [comment]
I also encountered the same problem. In my case, the page was stuck because a piece of code created tens of thousands of image previews
